app/api/views/reports/suppliers.py

```diff
@@ -59,3 +59,43 @@ def get_unassessed():
         result.append(i)
 
     return jsonify(result)
+
+
+@api.route('/reports/suppliers/all', methods=['GET'])
```

Comment:
The other two endpoints are singular and not plural - should this be `/reports/supplier/all` ?

Reply:
done